doc/design/volumes.md

```diff
@@ -39,7 +39,7 @@ type Volume struct {
 // VolumeStorer is an interface to create, remove, enumerate, and get Volumes.
 type VolumeStorer interface {
 	// Creates a volume on the given volume store, of the given size, with the given metadata.
-	VolumeCreate(ctx context.Context, ID string, store *url.URL, capacityMB uint64, info map[string][]byte) (*Volume, error)
+	VolumeCreate(ctx context.Context, ID string, store *url.URL, capacityKB uint64, info map[string][]byte) (*Volume, error)
```

Comment:
The type gets obfuscated by swagger in this case, so creating a `KB` type will require a swagger model and then a named parameter that the user will need to set.  Eventually we devolve to using names in one way (the model) or another (the parameter name).